.claude/commands/resume-worktrees.md

```
@@ -0,0 +1,86 @@
+---
+description: Re-open VS Code Insiders windows for every incomplete OloEngine worktree not already open (path-derived)
+argument-hint: <branch-slug, optional — omit to resume ALL incomplete worktrees>
+---
+Re-open the editor windows for in-progress OloEngine worktrees — e.g. after a reboot
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)